metricshub-doc/src/site/markdown/configuration/configure-monitoring.md

```diff
@@ -255,12 +255,52 @@ resourceGroups:
             password: mypwd
 ```
 
+#### JDBC
+
+Use the parameters below to configure the SQL protocol:
```

Comment:
SQL is not a protocol, please change to 
`Use the parameters below to configure JDBC to connect to a database:`

Reply:
@SafaeAJ , @NassimBtk : since this is related to the user doc, I will make the change required.

Reply:
@NassimBtk : I took your comment into account.
I will write the use case through a distinct issue. I am fine with the doc right now. You can proceed and you will be able to merge when you are fine with the code :)